src/frameworks/vite/index.ts

```diff
@@ -88,15 +88,18 @@ export async function build(root: string, target: string) {
   process.chdir(root);
 
   const originalNodeEnv = process.env.NODE_ENV;
-  // @ts-expect-error - NODE_ENV is `development` when building for production
-  // temporarily replace with target during build
-  process.env.NODE_ENV = target;
+
+  const envKey: string = "NODE_ENV";
```

Comment:
nit: mind adding a comment here to explain why `: string` is necessary?

Reply:
Done